src/components/Footer/Footer.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import './Footer.css';
+
+export const Footer = ({
+  todos,
+  filteredActive,
+  filteredCompleted,
+  handleReset,
+  handleClearCompleted,
+  indexTab,
+}) => (
+  <footer className="footer" style={{ display: 'block' }}>
+    <span className="todo-count">
+      {todos.filter(elem => elem.completed === false).length} items left
+    </span>
+
+    <ul className="filters">
+      <li onClick={handleReset}>
+        <a href="#/" className={indexTab === false ? 'selected' : ''}>
+          All
+        </a>
+      </li>
+
+      <li onClick={filteredActive}>
+        <a href="#/active" className={indexTab === 'active' ? 'selected' : ''}>
+          Active
+        </a>
+      </li>
+
+      <li onClick={filteredCompleted}>
```

Comment:
for onClick action you can create one general function with different cases, no need to create separate func for each item of the list